src/Report.php

```diff
@@ -83,7 +83,7 @@ public function __invoke($from, $to) {
         return $this->results();
     }
     
-    public function __construct(Event $event) {
+    public function __construct(EventBuilder $event) {
```

Comment:
__[code_sniffer]__
* Opening brace of a class must be on the line after the definition